src/jedi_bundle/config/bundles/build-order.yaml

```diff
@@ -70,8 +70,8 @@
     default_branch: develop
 - fv3-jedi:
     default_branch: develop
-- jedi-bundle:
-    default_branch: develop
+#- jedi-bundle:
+#    default_branch: develop
```

Comment:
Commented code should be removed, unless it has a good reason for sticking around, in which case it should be commented.

```suggestion
```